### ISSUE TITLE
During Exclude if the Result is "None" then TypeError is Raised 

### ISSUE THREAD

**Issue by abhinavvaidya90** [rheinwerk-verlag/postgresql-anonymizer#14]
During the exclude of rows, if a column returns _None_ value then the row[None] raises Type error

https://github.com/rheinwerk-verlag/postgresql-anonymizer/blob/5f6d7b3e1a9f4ae22e843eb1c6d57314a1939936/pganonymizer/utils.py#L101

```
Traceback (most recent call last):
  File "/home/ubuntu/.local/bin/pganonymize", line 11, in <module>
    sys.exit(main())
  File "/home/ubuntu/.local/lib/python3.6/site-packages/pganonymizer/__main__.py", line 10, in main
    main()
  File "/home/ubuntu/.local/lib/python3.6/site-packages/pganonymizer/cli.py", line 71, in main
    anonymize_tables(connection, schema.get('tables', []), verbose=args.verbose)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/pganonymizer/utils.py", line 38, in anonymize_tables
    data, table_columns = build_data(connection, table_name, columns, excludes, total_count, verbose)
  File "/home/ubuntu/.local/lib/python3.6/site-packages/pganonymizer/utils.py", line 68, in build_data
    if not row_matches_excludes(row, excludes):
  File "/home/ubuntu/.local/lib/python3.6/site-packages/pganonymizer/utils.py", line 101, in row_matches_excludes
    if pattern.match(row[column]):
TypeError: expected string or bytes-like object
```

Ref: myschema.yml
```
tables:
 - res_partner:
    fields:
     - name:
        provider:
          name: fake.name
     - email:
        provider:
          name: fake.email
    excludes: 
     - email:
        - "info.*@example.com"
```

Expected Behaviour :
- To ignore such records as it does not match the pattern any way

**Comment by hkage** [comment]
I cannot reproduce the traceback using a virtualenv with Python 3.8 on Ubuntu and a sample database I used for developing the pganonymizer.

Is `email` a column that is not present in the `res_partner` table or does the regex pattern simply doesn't match any data within the table?

**Comment by abhinavvaidya90** [comment]
For demo reason, I created a db with 1 table with 3 row
Table :

![image](https://user-images.githubusercontent.com/24862587/105481418-4f05b500-5ca7-11eb-9eaf-1c7343329018.png)

this is my .yml
![image](https://user-images.githubusercontent.com/24862587/105481608-94c27d80-5ca7-11eb-9a2e-94dff30dee85.png)

Result on running script
![image](https://user-images.githubusercontent.com/24862587/105481741-c2a7c200-5ca7-11eb-97aa-c8379acefb97.png)

**Comment by hkage** [comment]
The fix will be included in the new bugfix release 0.3.2.
